File: WiTrack/app/src/main/java/util/Device.java
package util;

/**
 * Created by j388923r on 1/5/16.
 */
public class Device {
    public boolean realtime_access;
    public int id, setup_id;
    public String setup_title, title;

    public Device(String title, String setup_title, int id, int setup_id, boolean realtime_access) {
        this.title = title;
        this.setup_title = setup_title;
        this.id = id;
        this.setup_id = setup_id;
        this.realtime_access = realtime_access;
    }
}

File: WiTrackApp/WiTrackApp/Device.swift
//
//  Device.swift
//  WiTrackApp
//
//  Created by Jamar Brooks on 12/7/15.
//  Copyright © 2015 Jamar. All rights reserved.
//

import Foundation

class Device {
    var id: Int!
    var realtime_access: Int!
    var setup_id: Int!
    var setup_title: String!
    var title: String!
    
    init(id: Int, realtime_access: Int, setup_id: Int, setup_title: String, title: String){
        self.id = id
        self.realtime_access = realtime_access
        self.setup_id = setup_id
        self.setup_title = setup_title
        self.title = title
    }
}